README.md

```diff
@@ -15,17 +15,20 @@ A curated list of awesome Fiber middlewares, boilerplates, recipes, articles and
 ## Contents
 <!--lint disable awesome-toc-->
 <!--lint disable awesome-git-repo-age-->
-- [⚙️ Middlewares](#%EF%B8%8F-middlewares)
-  - [🧬 Core](#-core)
-  - [🔗 External](#-external)
-  - [💻 Contrib](#-contrib)
-  - [🌱 Third Party](#-third-party)
-- [🚧 Boilerplates](#-boilerplates)
-- [📁 Recipes](#-recipes)
-- [🛠️ Tools](#%EF%B8%8F-tools)
-- [📖 Articles](#-articles)
-- [📺 Videos](#-videos)
-- [🤖 Benchmarks](#-benchmarks)
+- [Awesome Fiber ](#awesome-fiber)
+  - [Contents](#contents)
+  - [⚙️ Middlewares](#️-middlewares)
+    - [🧬 Core](#-core)
+    - [🔗 External](#-external)
+    - [‍💻 Contrib](#-contrib)
+    - [🌱 Third Party](#-third-party)
+  - [🚧 Boilerplates](#-boilerplates)
+  - [📁 Recipes](#-recipes)
+  - [🛠️ Tools](#️-tools)
+  - [📖 Articles](#-articles)
+  - [📺 Videos](#-videos)
+  - [🤖 Benchmarks](#-benchmarks)
+    - [👍 Contributing](#-contributing)
```

Comment:
No need to add contributing to here

Reply:
This one was added automatically. I've already removed it.